CesiumGeospatial/include/CesiumGeospatial/S2CellBoundingVolume.h

```diff
@@ -103,6 +108,8 @@ class CESIUMGEOSPATIAL_API S2CellBoundingVolume final {
   BoundingRegion computeBoundingRegion() const noexcept;
 
 private:
+  Ellipsoid _ellipsoid;
```

Comment:
Same thing that I mentioned in BoundingRegion here: adding this field this drastically increases the size of this type, so we should try really hard to avoid it.